tests/Strategies/Responses/UseApiResourceTagsTest.php

```diff
@@ -77,12 +86,18 @@ public function can_parse_apiresource_tags_with_model_factory_states()
     {
         $config = new DocumentationConfig([]);
 
+        $route = Mockery::mock(Route::class);
```

Comment:
Can you add a comment explaining why we have this?

Reply:
Because if you look at these two Api Resources Knuckles\Scribe\Tests\Fixtures\TestUserApiResource (lines 27-29) and Knuckles\Scribe\Tests\Fixtures\TestUserApiResourceCollection (lines 25-27). They now call the $request->route() method:

if($request->route()->named('test')) {
    $data['test'] = true;
}
I thought that mocking the request was a good way to test that if your logic includes calling the $request object inside an APIResource, the package instantiate the resource with all the necessary data.

Reply:
I meant adding a comment in the code, so the next person who sees it can know, but I'll handle that.